README.md

```diff
@@ -71,84 +77,78 @@ Now, we use a JSON configuration (Note you should be able to modify to use a dif
      -d '{"program.args": "/data/input.txt /data/output.dir"}'
 ```
 
-Few things to note here
+The status of the application can be monitored through the UI. Once the Spark application finishes running the output will be available in the output directory specified.
 
-* As the CDAP application template is wrapping it in ```NotifiableWorkflow```, you start the workflow. 
-* You pass the runtime argument as POST body.
-* And because it's a workflow it can scheduled to run periodically based on the set schedule. 
+## Advanced Options
 
-> **We are done, it was quick & easy to integrate your existing application into CDAP.**
+Now that the Spark program is running as a CDAP Application, users can leverage the additional capabilities provided by the CDAP platform such as - scheduling, logs, metadata, security and the like. The following section describes scheduling capability. 
 
-## Optional - Working with Schedules - Create, Update & List schedule for your application
+## Working with Schedules - Create, Update & List schedule for your application
 
 ### Create a Schedule for your Spark application
 
-Following shows how a schedule can be set to trigger a run of your Spark application at 4:00 AM
+Create a daily schedule to trigger a run of your Spark application at 4:00 AM every day using the following REST API:
 
 ```
-curl -w"\n" -X PUT "localhost:11015/v3/namespaces/default/apps/WordCountApp/schedules/DailySchedule" \
+curl -w"\n" -X PUT "localhost:11015/v3/namespaces/default/apps/WordCount/schedules/DailySchedule" \
      -d  '{ "scheduleType": "TIME", "program": { "programName": "NotifiableWorkflow", "programType": "WORKFLOW" }, "properties": {}, "schedule": { "cronExpression": "0 4 * * *", "name": "DailySchedule"} }'
 ```
 
-Few important things to notice, in the POST body, the configuration for the schedule is defined in there. So, let's look deeper into what's being specified as configuration for adding a new schedule.
+The POST body contains the configuration for the schedule created. Details as follows:
 
-* ```scheduleType``` This defines the type of schedule you are attempting to set, there are different kinds supported within CDAP, but, for this application, we only support ```TIME``` based. 
-* ```programName``` Specifies the CDAP Program that wraps your spark application. 
-* ```schedule``` Specifies the [crontab expression](http://www.adminschoice.com/crontab-quick-reference) to schedule and the name of the schedule. If the schedule with the same name is already present, the REST API call will fail. In that case, please use update to modify the schedule parameters. 
+* ```scheduleType``` This defines the type of schedule you are attempting to set. 
+* ```programName``` Specifies the CDAP Program that runs the Spark plugin. 
+* ```schedule``` Specifies the [cron expression](http://www.adminschoice.com/crontab-quick-reference) for the schedule and the name of the schedule. 
```

Comment:
maybe better link would be `http://www.quartz-scheduler.org/documentation/quartz-2.1.x/tutorials/crontrigger.html `